lib/resources/users.js

```diff
@@ -55,7 +55,7 @@ module.exports = (service, endpoint) => {
         .orElseGet(() => Users.emailEverExisted(body.email)
           .then((existed) => ((existed === true)
             ? mail(body.email, 'accountResetDeleted')
-            : mail(body.email, 'accountResetFailure'))))
```

Comment:
With this gone, I think we can go ahead and remove this message in `lib/formats/mail.js`.